src/js/server/devpmt-base-dispatchers.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Base Dispatchers
+ *
+ * Contains `gpii.devpmt.baseDispatcher` which can be used for building
+ * the specific dispatchers in the system. Brings in functionality from
+ * gpii-express and gpii-handlebars and adds extra hooks for performing
+ * authorization checks and adding additional context to the rendering
+ * contexts.
+ *
+ * Copyright 2019 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+/* eslint-env node */
+"use strict";
+var path = require("path");
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+
+/**
+ * gpii.devpmt.baseDispatcher - The simplest base dispatcher that
+ * has common features such as our templates.
+ *
+ * This dispatcher container an invoker `contextPromise` that can
+ * be overridden to add extra keys to the context that is given to
+ * the handlebars renderer.
+ */
+fluid.defaults("gpii.devpmt.baseDispatcher", {
+    gradeNames: ["gpii.express.middleware.requestAware", "gpii.handlebars.dispatcherMiddleware"],
+    method: "get",
+    templateDirs: ["@expand:fluid.module.resolvePath(%gpii-devpmt/src/templates)"],
```

Comment:
Given you throw them into gpii.express.hb.resolveAllPaths https://github.com/GPII/gpii-handlebars/blob/master/src/js/server/lib/resolver.js#L24 this can just accept the module paths directly
 - e.g. templateDirs: ["%gpii-devpmt/src/templates"] -

Reply:
that's a nice simplification